tests/integration/views/test_questionnaire.py

```diff
@@ -66,12 +68,14 @@ def test_update_questionnaire_store_with_default_value(self):
             'answer': None
         }
 
-        with self._application.test_request_context():
+        with self._application.test_request_context(), patch('app.helpers.form_helper.uuid4') as uuid:
+            uuid.return_value = '1'
```

Comment:
`patch('app.helpers.form_helper.uuid4', return_value='1')` is cleaner way to patch this considering you're not asserting on uuid